prime-router/src/main/kotlin/Report.kt

```diff
@@ -569,6 +569,14 @@ class Report : Logging {
                     it.siteOfCare = row.getStringOrNull("site_of_care").trimToNull()
                     it.reportId = this.id
                     it.reportIndex = idx
+                    // For sender ID, use first the provided ID and if not use the client ID.
+                    it.senderId = row.getStringOrNull("sender_id").trimToNull()
+                    if (it.senderId.isNullOrBlank()) {
+                        val clientSource = sources.firstOrNull { source -> source is ClientSource } as ClientSource?
+                        if (clientSource != null) it.senderId = "${clientSource.organization}.${clientSource.client}"
```

Comment:
Is there a Sender.fullName field that give you the "orgName.clientName" format?

Reply:
Nope.  A good idea to add it.  Done.